.github/workflows/create-reports.yml

```diff
@@ -61,6 +65,7 @@ jobs:
     
     - name: Rebase from develop
       run: |
+        git config --global core.longpaths true
```

Comment:
Even with the short folder name we got some >260 chars paths of generated images, which git on Windows can only handle with this option.
@pchelle Please add the same in the PBPK-Library